sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_v1/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+friendly_name: Aggregation for Active Users
+description: |-
+  Active users aggregated by Mozilla product and business dimensions
+  (attribution details, channel, country, city, date, device model,
+  distribution_id, os details, segment)
+  Proposal:
+  https://docs.google.com/document/d/1qvWO49Lr_Z_WErh3I3058A3B1YuiuURx19K3aTdmejM/edit?usp=sharing
+owners:
+- lvargas@mozilla.com
+labels:
+  incremental: true
+scheduling:
+  dag_name: bqetl_analytics_aggregations
+  task_name: active_users_aggregates_v1
+bigquery:
+  time_partitioning:
+    field: submission_date
+    type: day
+    require_partition_filter: true
+    expiration_days: null
+  clustering:
+    fields:
+    - country
+    - app_name
+    - attribution_medium
+    - channel
+references: {}
```

Comment:
One more thing that occurred to me is that we should probably add a few more info to the `references` section to set the dependency between this DAG and the unified metrics one (we want to make sure this only runs after unified metrics derived table is updated).

Check out this example:
https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-marketing-prod/acoustic/contact_raw_v1/metadata.yaml#L22-L31

Reply:
That sounds like a requirement. I would like to delay this dependency for a few runs until the process and data is stable end-to-end. Also would a notification of outdated data be an option to the dependency?

Reply:
Sure, we can leave it for now. Not sure I got the second question. In this case, if this DAG would run before the `unified_metrics` one for the same execution date there would just no data in this table for the submission_date.

Reply:
but actually, interestingly I don't see how the task dependency has been defined. But it looks like it already exists when generating the DAG:
[telemetry_derived__unified_metrics__v1](https://github.com/mozilla/bigquery-etl/pull/2952/files#diff-4cb089996f5af781b0342647073e4b030aa633bcc5b93ab7b4fc9f1d97dd5f87R63)

🤷